### PR TITLE
Hammerdb update mem

### PR DESCRIPTION
**Pull request by ebattat**

### CODE REVIEW

File: benchmark_runner/benchmark_operator/workload_flavors/perf_ci/hammerdb/hammerdb_data_template.yaml [hunk 1]
```diff
@@ -19,9 +19,15 @@ shared_data:
...
   mariadb_limits_cpu: 32
+<<<<<<< HEAD
+  mariadb_limits_memory_GB: 32
+  mariadb_requests_cpu: 32
+  mariadb_requests_memory_GB: 32
+=======
   mariadb_limits_memory_GB: 64
   mariadb_requests_cpu: 32
   mariadb_requests_memory_GB: 64
+>>>>>>> main
```
Comment (RobertKrawitz) [inline benchmark_runner/benchmark_operator/workload_flavors/perf_ci/hammerdb/hammerdb_data_template.yaml:30]: Conflict needs merging

File: benchmark_runner/benchmark_operator/workload_flavors/perf_ci/hammerdb/hammerdb_data_template.yaml [hunk 2]
```diff
@@ -30,9 +36,15 @@ mariadb:
...
   mssql_limits_cpu: 32
+<<<<<<< HEAD
+  mssql_limits_memory_GB: 32
+  mssql_requests_cpu: 32
+  mssql_requests_memory_GB: 32
+=======
   mssql_limits_memory_GB: 64
   mssql_requests_cpu: 32
   mssql_requests_memory: 64
+>>>>>>> main
```
Comment (RobertKrawitz) [inline benchmark_runner/benchmark_operator/workload_flavors/perf_ci/hammerdb/hammerdb_data_template.yaml:47]: Conflict needs merging

File: benchmark_runner/benchmark_operator/workload_flavors/perf_ci/hammerdb/hammerdb_data_template.yaml [hunk 3]
```diff
@@ -41,9 +53,15 @@ mssql:
...
   postgres_limits_cpu: 32
+<<<<<<< HEAD
+  postgres_limits_memory_GB: 32
+  postgres_requests_cpu: 32
+  postgres_requests_memory_GB: 32
+=======
   postgres_limits_memory_GB: 64
   postgres_requests_cpu: 32
   postgres_requests_memory_GB: 64
+>>>>>>> main
```
Comment (RobertKrawitz) [inline benchmark_runner/benchmark_operator/workload_flavors/perf_ci/hammerdb/hammerdb_data_template.yaml:64]: Conflict needs merging

File: benchmark_runner/benchmark_operator/workload_flavors/perf_ci/hammerdb/hammerdb_data_template.yaml [hunk 4]
```diff
@@ -59,5 +77,10 @@ pod:
 vm:
   vm_cores: 1
   vm_sockets: 32
+<<<<<<< HEAD
+  vm_limits_memory_GB: 32
+  vm_requests_memory_GB: 32
+=======
   vm_limits_memory_GB: 64
   vm_requests_memory_GB: 64
+>>>>>>> main
```
Comment (RobertKrawitz) [inline benchmark_runner/benchmark_operator/workload_flavors/perf_ci/hammerdb/hammerdb_data_template.yaml:86]: Conflict needs merging

**Review by RobertKrawitz — CHANGES_REQUESTED** [review]
Conflicts need to be merged.

**Comment by ebattat** [comment]
Conflicts problems

**Pull request closed without merging**